js/popup.js

```diff
@@ -0,0 +1,176 @@
+import projectCardAttributes from './attributes.js';
+
+const popUpCardContainerNew = document.querySelector('.popup-card-container');
+
+const generatePopupProjectCard = (
+  imgSrc,
+  imgTitle,
+  imgAlt,
+  cardDescription,
+  tags,
+  btnHrefOne,
+  btnHrefTwo,
+  popupNavPrevious,
+  popupNavNext,
```

Comment:
Let's make this popupNavPreviousId and popupNavNextId instead 

```suggestion
  popupNavPreviousId,
  popupNavNextId,
```